resources/schemas/dbscripts/postgresql/targetedms-19.25-19.26.sql

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE TABLE targetedms.ListDefinition
+(
+    Id SERIAL NOT NULL,
+    RunId INT NOT NULL,
+    Name TEXT NOT NULL,
+    PkColumnIndex INT NULL,
+    DisplayColumnIndex INT NULL,
+    CONSTRAINT PK_List PRIMARY KEY(Id),
+    CONSTRAINT FK_List_RunId FOREIGN KEY(RunId) REFERENCES targetedms.Runs(Id)
+);
+CREATE TABLE targetedms.ListColumnDefinition
+(
+    Id SERIAL NOT NULL,
+    ListDefinitionId INT NOT NULL,
+    ColumnIndex INT NOT NULL,
+    AnnotationType VARCHAR(20) NOT NULL,
+    Name TEXT NOT NULL,
```

Comment:
Does this really need to be unlimited? I see the code that is conditional on SQL Server for this. Does anyone need list columns that are more than a few hundred characters? Do these get autogenerated?